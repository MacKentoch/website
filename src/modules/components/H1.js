import styled from 'styled-components'

const H1 = styled.h1`
  font-size: 40px;
  line-height: 1.2;
  margin: 10px 0;
  font-weight: 400;
  @media (min-width: 700px) {
    font-size: 50px;
  }
`

export default H1
