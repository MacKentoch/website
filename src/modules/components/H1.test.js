import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import H1 from './H1'

describe('H1', () => {
  it('should render a h1', () => {
    const wrapper = shallow(<H1>Hello</H1>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
