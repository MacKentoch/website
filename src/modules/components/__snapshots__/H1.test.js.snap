// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 should render a h1 1`] = `
<h1
  className="sc-bdVaJa hqVBEH"
>
  Hello
</h1>
`;
